selfdrive/ui/qt/widgets/drive_stats.cc

```diff
@@ -66,5 +65,5 @@ DriveStats::DriveStats(QWidget* parent) : QWidget(parent) {
   QString dongleId = QString::fromStdString(Params().get("DongleId"));
   QString url = "https://api.commadotai.com/v1.1/devices/" + dongleId + "/stats";
   RequestRepeater *repeater = new RequestRepeater(this, url, "ApiCache_DriveStats", 13);
-  QObject::connect(repeater, SIGNAL(receivedResponse(QString)), this, SLOT(parseResponse(QString)));
+  QObject::connect(repeater, &RequestRepeater::receivedResponse, this, &DriveStats::parseResponse);
```

Comment:
nice, we should start moving all our signal/slots to this syntax

Reply:
working on that